smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/readwrite/ResponseClosureUtils.kt

```diff
@@ -0,0 +1,26 @@
+package software.amazon.smithy.swift.codegen.integration.serde.readwrite
+
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.swift.codegen.SwiftWriter
+import software.amazon.smithy.swift.codegen.integration.ProtocolGenerator
+
+class ResponseClosureUtils(
```

Comment:
Generates a reference to the HTTP response closure for an output shape.
- For XML, these are code-generated.
- For JSON, the closure comes from runtime.